ui/src/testing/stubs/surveys-api-stub.ts

```diff
@@ -1,20 +1,14 @@
 import { CreateNewUserSatisfactionSurvey, SurveysApi } from 'generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class SurveysApiStub extends SurveysApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.